acra/guides/trying-acra-with-docker/_index.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Trying Acra with Docker
+bookCollapseSection: true
+---
+
+# Acra + Docker
+
+You can use containers with Acra. Acra works well with [Docker](https://www.docker.com/what-docker). However, we advise only going the "Acra in Docker containers" route for testing purposes, and NOT in a real-world production environment.
+
+This is why this document is called "Trying Acra with Docker", not "Using Acra with Docker in a Production Setting".
+
+### Precautions
+
+Storing keys safely when using Docker is problematic. Docker is immutable while [Zones]({{< ref "acra/acra-in-depth/cryptography-and-key-management#zones-INVALID" >}})/[keys]({{< ref "acra/acra-in-depth/cryptography-and-key-management#-INVALID" >}}) are not. This means that you might want to attach some storage and end up making the keys accessible to attackers. There are multiple ways to solve this problem and it will be addressed in a convenient and secure fashion in the future releases of Acra.
```

Comment:
Actually not. If it's a server where running only Acra then it's okay. But if it's some server where deployed different containers and you store keys on the host layer and mount it to Acra containers - it's not the same and not safe. Because this folder may be mounted to other containers. Or some container are vulnerable, can escalate permissions and get access to root FS. So keys are not safe in such cases that we can't ignore. And we can't check or handle it from Acra's containers.